src/v2/guide/render-function.md

```diff
@@ -343,14 +343,15 @@ render: function (createElement) {
     },
     on: {
       input: function (event) {
-        self.value = event.target.value
         self.$emit('input', event.target.value)
       }
     }
   })
 }
 ```
 
+_Note_: In the example above we consider a prop named `value` already exists.
```

Comment:
What about just add `props: ['value']` above `render` function?

Reply:
The code is just showing the render function, adding props here would make it inconsistent with the other samples